src/tests/multihost/alltests/conftest.py

```diff
@@ -424,6 +424,33 @@ def restore_sssd_conf():
     request.addfinalizer(restore_sssd_conf)
 
 
+@pytest.fixture(scope='function')
+def sssd_sudo_conf(session_multihost, request):
+    """ Configure basic sudo parameters in sssd.conf """
+    tools = sssdTools(session_multihost.client[0])
+    session_multihost.client[0].service_sssd('stop')
+    tools.remove_sss_cache('/var/lib/sss/db/')
+    tools.remove_sss_cache('/var/log/sssd')
+    ldap_uri = 'ldap://%s' % session_multihost.master[0].sys_hostname
```

Comment:
Do we have (or should we set) a string formatting standard to use?  In other parts of conftest I see this style as well as the newer str.format style.  I'm not against using this style.  Just that we should try to be consistent across our test code.

Reply:
I agree with your point. There should be a formatting standard to adhere. The ldap_uri has been implemented in three ways:
1.  ldap_uri = 'ldaps://%s' % (ds_host)
2. ldap_uri = 'ldap://%s' % (session_multihost.master[0].sys_hostname)
3. ldap_uri = 'ldaps://%s' % session_multihost.master[0].sys_hostname

What is your opinion about preferred way of formatting?  I find first one useful.

Reply:
Oh I didn't catch that.   There I think either 1 or 3 is fine depending on line length where you may need to use 1 to stay within limits.  I was talking about the old style:

ldaps://%s' % session_multihost.master[0].sys_hostname

vs: the newer style using string format():

'ldaps://{}'.format(session_multihost.master[0].sys_hostname)

vs the python 3.6 f-strings:

f'ldaps://{session_multihost.master[0].sys_hostname}'

Personally I prefer the newer f-strings because it usually makes line length shorter and the code easier to read (for me at least).

Reply:
After implementing f-strings, code-block looks better.